pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -39,78 +39,83 @@
-    click.option('--filter-query-language', type=str, required=False,
+                 help='Prompt for selecting an instance from a list. '
+                      'If used, the INSTANCENAME argument must be a class '
+                      'name, and the instances of that class are presented.')]
+
+property_create_option = [              # pylint: disable=invalid-name
+    click.option('-P', '--property', type=str, metavar='PROPERTYNAME=VALUE',
+                 required=False, multiple=True,
+                 help='Initial property value for the new instance. '
+                      'May be specified multiple times. '
+                      'Array property values are specified as a '
+                      'comma-separated list; embedded instances are not '
+                      'supported. '
+                      'Default: No initial properties provided.')]
+
+property_modify_option = [              # pylint: disable=invalid-name
+    click.option('-P', '--property', type=str, metavar='PROPERTYNAME=VALUE',
+                 required=False, multiple=True,
+                 help='Property to be modified, with its new value. '
+                      'May be specified multiple times. '
+                      'Array property values are specified as a '
+                      'comma-separated list; embedded instances are not '
+                      'supported. '
+                      'Default: No properties modified.')]
+
+filter_query_language_option = [              # pylint: disable=invalid-name
+    click.option('--filter-query-language', type=str, metavar='QUERY-LANGUAGE',
                  default=None,
-                 help='A filter-query language to be used with a filter query '
-                 'defined by --filter-query. (Default: None)')]
+                 help='The filter query language to be used with '
+                      '--filter-query. '
+                      'Default: DMTF:FQL.')]
 
-filterquery_option = [              # pylint: disable=invalid-name
-    click.option('-f', '--filter-query', type=str, required=False,
+filter_query_option = [              # pylint: disable=invalid-name
+    click.option('-f', '--filter-query', type=str, metavar='QUERY-STRING',
                  default=None,
-                 help='A filter query to be passed to the server if the pull '
-                 'operations are used. If this option is defined and the '
-                 '--filter-query-language is None, pywbemcli assumes DMTF:FQL. '
-                 'If this option is defined and the traditional operations are '
-                 'used, the filter is not sent to the server. See the '
-                 'documentation for more information. (Default: None)')]
+                 help='When pull operations are used, filter the instances in '
+                      'the result via a filter query. '
+                      'By default, and when traditional operations are used, '
+                      'no such filtering takes place.')]
```

Comment:
Who cares what is sent to the server? We should describe the behavior that is visible to users of pywbemcli, not what we do to achieve that, leaving the conclusions about what that means for them.l to the reader.

Reply:
Not good, because it is only right for the case of traditional operations, but not for the default case of not specifying the option with pull operations.